weaviate/collections/classes/grpc.py

```diff
@@ -153,13 +154,39 @@ class GroupBy(_WeaviateInput):
     number_of_groups: int
 
 
-class Sort(_WeaviateInput):
-    """Define how the query's sort operation should be performed."""
-
+class _Sort(_WeaviateInput):
     prop: str
     ascending: bool = Field(default=True)
 
 
+class Sort(_Sort):
+    """Define how the query's sort operation should be performed using the available static methods."""
+
+    def __init__(self, prop: str, ascending: bool = True) -> None:
+        _Warnings.sort_init_deprecated()
+        super().__init__(prop=prop, ascending=ascending)
+
+    @staticmethod
+    def by_property(prop: str, order: Literal["asc", "desc"] = "asc") -> _Sort:
+        """Sort by a property."""
+        return _Sort(prop=prop, ascending=order == "asc")
+
+    @staticmethod
+    def by_id(order: Literal["asc", "desc"] = "asc") -> _Sort:
```

Comment:
should we allow chaining?
Sort.by_property("age").by_creation_time()

Reply:
As an alternative to providing `List[Sort]`? I think that would be a good feature

Reply:
exactly